tests/scripts/remote_monitoring_tests/rest_apis/test_list_recommendations.py

```diff
@@ -1054,7 +1070,9 @@ def test_list_recommendations_for_diff_reco_terms_with_only_latest(test_name, nu
         exp_found = False
         for list_reco in list_reco_json:
             if create_exp_json[0]['experiment_name'] == list_reco['experiment_name']:
-                validate_reco_json(create_exp_json[0], update_results_json, list_reco, expected_duration_in_hours, test_name)
+                plots_available = False
```

Comment:
@khansaad Setting this flag will not validate the terms that have recommendations.

Reply:
Removed the flag, able to run the tests successfully.